config.js

```diff
@@ -258,7 +258,11 @@ config.iota = {
      * Default resource of the IoT Agent. This value must be different for every IoT Agent connecting to the IoT
      * Manager.
      */
-    defaultResource: '/iot/json'
+    defaultResource: '/iot/json',
+    /**
+     * flag indicating whether the incoming measures to the IoTAgent should be processed as per the "attributes" field.
+     */
+    explicitAttrs: true
```

Comment:
```suggestion
    explicitAttrs: false
```

In order to keep backward compatibility in previous versions of the IOTA.

Reply:
Fixed in f8d801e